src/account.js

```diff
@@ -149,6 +149,17 @@ export class MuxedAccount {
     return new MuxedAccount(new Account(gAddress, sequenceNum), id);
   }
 
+  /**
+   * A helper method to turn an M-address into its underlying G-address.
+   *
+   * @param   {string} mAddress - muxed address to convert
+   * @returns {string} underlying G-address
+   */
+  static parseBaseAddress(mAddress) {
```

Comment:
The name of this function is a little confusing. Often functions named `parseX` are understood to "parse X". In this case the function is named `parseBaseAddress` but what it does is "parse an M address".

I think it would be easier to understand the intent of this function if it was named `parseMuxedAddress` and it returned all the components of a muxed address. I thought we would have had something like this already, but I guess we don't?

Reply:
I see what you're saying. We do have that: `MuxedAccount.fromAddress`, but doing M -> G strings becomes unwieldy:

```js
const muxed = new StellarBase.MuxedAccount.fromAddress(mAddress, '0');
const gAddress = muxed.baseAccount().accountId();
```

(see cfc6df0). How about `extract` over `parse`, @leighmcculloch? e.g. 

```js
const gAddress = MuxedAccount.extractBaseAddress(mAddress);
```